test/gurka/test_landmarks.cc

```diff
@@ -1,61 +1,128 @@
 #include <filesystem>
 #include <gtest/gtest.h>
+#include <vector>
 
 #include "gurka.h"
-#include "mjolnir/landmark_database_builder.h"
+#include "mjolnir/landmark_builder.h"
 #include "test/test.h"
+#include <boost/property_tree/ptree.hpp>
 
 using namespace valhalla;
 using namespace valhalla::baldr;
 using namespace valhalla::gurka;
 using namespace valhalla::mjolnir;
 
-const std::string db_name = "landmarks.db";
-const std::filesystem::path file_path = db_name;
+namespace {
+valhalla::gurka::map BuildPBF(const std::string& workdir) {
+  const std::string ascii_map = R"(
+      a-----b-----c---d
+      A   B   C   D   E
+    )";
 
-class LandmarkDatabaseTest : public ::testing::Test {
-protected:
-  static void SetUpTestSuite() {
-    LandmarkDatabase db(db_name, false);
+  const gurka::nodes nodes = {
+      {"A", {{"name", ""}, {"amenity", "bar"}}},
```

Comment:
in real life osm will not store a tag without a value, so if you want to test this then we should just omit the name tag completely:

```suggestion
      {"A", {{"amenity", "bar"}}},
```

Reply:
good to know!